.github/ISSUE_TEMPLATE/1_feature.yml

```diff
@@ -0,0 +1,47 @@
+name: Feature Request
+description: Create a feature request.
+labels:
+  - type::feature
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Processing feature requests is time-consuming so please fill out the following form as completely as possible. **The more complete this form is, the more likely we can provide a speedier response.**
+
+        > ⚠️ Requests that are incomplete or missing information will simply be closed.
+
+        Since there are already a lot of open requests, please take a moment to search existing ones to see if your idea has already been proposed. If you find something related, please upvote that request and provide additional details as necessary.
+
+        > 💐 Thanks for helping to make Conda better! We would be unable to do the things we do without our community!
```

Comment:
```suggestion
        Because processing new feature requests is time-consuming, we would like to ask you to fill out the following form to the best of your ability and as completely as possible.
        
        > **Note**
        > ⚠️ Feature requests that are incomplete or missing information may be closed as inactionable.

        Since there are already a lot of open issues, please also take a moment to search existing ones to see if your feature request has already been submitted. If you find something related, please upvote that issue and provide additional details as necessary.

        💐 Thank you for helping to make Conda better. We would be unable to improve Conda without our community!
```